p2-feature/pom.xml

```diff
@@ -48,6 +48,18 @@
             <plugin>
                 <groupId>org.eclipse.tycho</groupId>
                 <artifactId>tycho-maven-plugin</artifactId>
+                <dependencies>
+                    <dependency>
+                        <groupId>org.eclipse.tycho</groupId>
+                        <artifactId>tycho-core</artifactId>
+                        <version>${tycho-version}</version>
+                    </dependency>
+                    <dependency>
+                        <groupId>org.eclipse.tycho</groupId>
+                        <artifactId>tycho-p2-facade</artifactId>
+                        <version>${tycho-version}</version>
+                    </dependency>
+                </dependencies>
```

Comment:
The tycho stuff should probably be in its own review and be merged first, before new features?

Reply:
I can move it out. But they are a separate commit so thought might as well merge it with this PR. Let me know what do you prefer.